### DISCUSSION TITLE
Disabling GitHub Release + Publish

### DISCUSSION THREAD

**Discussion by myang310** [goreleaser/goreleaser#2570] [Q&A] [answered]
Is there any way to disable the GitHub release/publish steps?

**Comment by caarlos0** [comment, marked as answer]
There is a `disable` flag in the release config https://goreleaser.com/customization/release/

you can also run with `--skip-publish` https://goreleaser.com/cmd/goreleaser_release/

**Reply by myang310** [reply]
Can you disable it just for GitHub but still release/publish to Artifactory?

**Reply by caarlos0** [reply]
yes, first link I just posted.
